File: sub-mysql-web/src/main/java/org/github/base/service/impl/SysUserRoleServiceImpl.java
package org.github.base.service.impl;

import org.github.base.entity.SysUserRoleEntity;
import org.github.base.mapper.ISysUserRoleMapper;
import org.github.base.service.ISysUserRoleService;
import org.github.base.ServiceImpl;
import org.springframework.stereotype.Service;

/**
 * <p>
 * 用户与角色对应关系 服务实现类
 * </p>
 *
 * @author JYD_XL
 * @since 2021-05-20
 */
@Service
public class SysUserRoleServiceImpl extends ServiceImpl<ISysUserRoleMapper, SysUserRoleEntity> implements ISysUserRoleService {}

File: sub-mysql-mydb-service/src/main/java/org/github/mysql/mydb/base/service/impl/SysUserRoleServiceImpl.kt
@file:Suppress("RedundantSemicolon")

package org.github.mysql.mydb.base.service.impl;

import org.github.mysql.mydb.base.entity.SysUserRoleEntity;
import org.github.mysql.mydb.base.mapper.ISysUserRoleMapper;
import org.github.mysql.mydb.base.service.ISysUserRoleService;
import org.github.base.ServiceImpl;
import org.springframework.stereotype.Service;

/**
 * 用户-角色关系表 服务实现类
 *
 * @author JYD_XL
 */
@Service
open class SysUserRoleServiceImpl : ServiceImpl<ISysUserRoleMapper, SysUserRoleEntity>(), ISysUserRoleService
